content/docs/guides/scim/okta.md

```diff
@@ -0,0 +1,136 @@
+
+1. Under the **Provisioning** tab, click **Edit**.
+2. While you are in the **To App** section, check **Enable** on the following:
+
+    * Create Users
+    * Update User Attributes
+    * Deactivate Users
+3. Click **Save**.
+
+## Adjusting the Okta to Pulumi Attribute Mappings
+
+Now that the SCIM connector knows how to connect to Pulumi, you need to adjust the attributes that Okta will send to Pulumi and what it might receive back from Pulumi. To do that, perform the following steps:
+
+1. Under **Provisioning** > **To App**, click **Go to Profile Editor**.
+2. Click **Mappings** to open the Attribute Editor dialog box.
+3. Un-map all but the following attributes:
+
+    * givenName
+    * familyName
+    * displayName
+    * email
+
+4. Click on the **Okta User To …** tab and un-map all but the following attributes:
+
+    * givenName
+    * familyName
+    * displayName
+    * email
+
+5. Click **Save Mappings**.
+6. Click **Apply Updates**.
+
+## Setting up Group Provisioning
+
+The Pulumi console supports the provisioning of teams within your organization using SCIM. This is done by mapping the groups you have created using SCIM to create teams within your organization in the Pulumi console. Setting this up allows you to manage your teams' memberships solely in Okta.
+
+To set this up, you need to enable Push Groups as a supported provisioning action under the **Provisioning** settings and then specify which groups you would like to push. To do that, performing the following steps:
+
+1. Navigate to the application that you created above by clicking the on the **Applications** tab and then selecting the application.
+
+    ![Okta application selection.](/images/docs/guides/scim/okta-application-selection-scim.png)
+
+2. Click on **Provisioning** tab, then click on the integrations menu option located on the side navigation.
+3. Check the **Push Groups** box under **Supported provisioning actions**.
+4. Select the **Push Groups** tab.
+5. Select the **Push Groups** drop down menu. Here you will have the option to find groups either by name or rule.
+
+    ![Push Groups drop down menu in Okta](/images/docs/guides/scim/okta-pushgroups-menu-scim.png)
+
+6. Select **Find groups by name**. This will take you to a screen where you can search groups by the group name. Once you find the group(s) that you are looking for, click **Save**.
+
+    You will notice there is a checkbox labeled **Push group memberships immediately**. This will also add all the members that are part of your group to the Pulumi team after your selection has been saved.
+
+7. After clicking **Save**, you should see that your group was pushed and it should say **Active**.
+
+    Note: if there are members in a group that are not yet assigned to the Pulumi Console application in Okta, they will not be added to the team in the Pulumi Console. Ensure that all members in the group have been assigned to the application before pushing the group.
```

Comment:
Should this note contain the notes styling similar to how it is done on line 51?

Reply:
Yup, good catch. Included in https://github.com/pulumi/docs/pull/5139/commits/73b0e75c5d641c053a4fa585ceeb0101abb21dd7